src/WingetCreateCore/Common/GitHub.cs

```diff
@@ -200,6 +200,18 @@ public async Task<string> GetFileContentsAsync(string path)
         public async Task CheckAccess()
         {
             await this.github.Repository.Get(this.wingetRepoOwner, this.wingetRepo);
+        }
+
+        /// <summary>
+        /// Recursively searches the repository for the provided package identifer to determine if it already exists.
+        /// </summary>
+        /// <param name="packageId">The package identifier.</param>
+        /// <returns>Tuple value with T1 being a boolean that represents whether the inputted packageId matches an existing one
+        /// and T2 as the string of the matching, (case-sensitive) exact packageId. </returns>
+        public async Task<(bool IsMatch, string ExactId)> CheckDuplicatePackageId(string packageId)

```

Comment:
>CheckDuplicatePackageId [](http://example.com/codeflow?start=58&length=23)

Why not just call this FindPackageId or something and have it return the case-sensitive match if found, otherwise null

Reply:
I agree. Changed to suggestion. FindPackageId now returns the match if found, null otherwise.